### DISCUSSION TITLE
[Feature] I wish topology of  skywalking v9.0.0  can  have group  menu

### DISCUSSION THREAD

**Discussion by czxin788** [apache/skywalking#8893] [Q&A] [answered]
### Search before asking

- [x] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

At v8.x of skywalking, it has group menu and  create group menu:
![image](https://user-images.githubusercontent.com/13861904/163717477-b16b357a-071a-4143-8d36-9d300e7a6233.png)

But at v9.0.0 of skywalking，the group menu  have  lost:
![image](https://user-images.githubusercontent.com/13861904/163717591-a395a4ff-eaa2-407b-bc1c-06b85ef8128a.png)

### Use case

 I support   topology of  skywalking v9.0.0  can  have group  menu and create group menu.

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment, marked as answer]
Group in v8 has a different meaning from v9. It is replaced by **layer**. 

Don't use this like a tenancy label or a thing separating dev/prod env, this is never the purpose of SkyWalking v8's group concept.
